layers/+web-services/llm-client/packages.el

```diff
@@ -44,18 +44,18 @@
     :defer t
     :ensure t
     :init
-    ;; evilify gptel-context-buffer-mode-map
-    (require 'gptel-context)
-    (evil-set-initial-state 'gptel-context-buffer-mode 'evilified)
-    (evilified-state-evilify-map gptel-context-buffer-mode-map
-      :mode gptel-context-buffer-mode
-      :bindings
-      "C-c C-c" #'gptel-context-confirm
-      "C-c C-k" #'gptel-context-quit
-      "RET"     #'gptel-context-visit
-      "n"       #'gptel-context-next
-      "p"       #'gptel-context-previous
-      "d"       #'gptel-context-flag-deletion)
+    (with-eval-after-load 'gptel-context
+      ;; evilify gptel-context-buffer-mode-map
+      (evil-set-initial-state 'gptel-context-buffer-mode 'evilified)
+      (evilified-state-evilify-map gptel-context-buffer-mode-map
```

Comment:
`evilified-state-evilify-map` has an `:eval-after-load` argument which can be used to do this a little more concisely, if you'd like.